kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/test/java/org/kie/server/services/jbpm/ui/ImageServiceBaseTest.java

```diff
@@ -201,6 +201,112 @@ public void testLoopSubProcess() throws Exception {
+
+        NodeInstanceDesc nodeInstanceDescCompleted3 = new org.jbpm.kie.services.impl.model.NodeInstanceDesc("3", twoScriptNode, "", "ScriptNode", "", 1L, null,
+                                                                                                            "", 0, 1L, 4L, "", null, 1);
+
+        NodeInstanceDesc nodeInstanceDescError = new org.jbpm.kie.services.impl.model.NodeInstanceDesc("4", threeScriptNode, "", "ScriptNode", "", 1L, null,
+                                                                                                       "", 6, 1L, 1L, "", null, 1);
+
+        List<NodeInstanceDesc> activeNodes = Collections.EMPTY_LIST;
+        List<NodeInstanceDesc> completedNodes = Arrays.asList(nodeInstanceDescCompleted1, nodeInstanceDescCompleted2);
+        List<NodeInstanceDesc> fullLogs = Arrays.asList(nodeInstanceDescCompleted1, nodeInstanceDescCompleted2, nodeInstanceDescCompleted3, nodeInstanceDescError);
+        when(dataService.getProcessInstanceHistoryCompleted(anyLong(), any())).thenReturn(completedNodes);
+        when(dataService.getProcessInstanceHistoryActive(anyLong(), any())).thenReturn(activeNodes);
+        when(dataService.getProcessInstanceFullHistory(anyLong(), any())).thenReturn(fullLogs);
+        Map<String, ImageReference> imageReferenceMap = new HashMap<>();
+        imageReferenceMap.put(containerId, imageReference);
+
+        ProcessInstanceDesc processInstanceDesc = new org.jbpm.kie.services.impl.model.ProcessInstanceDesc(processInstanceId, processId, "", "", 1, containerId, null
+                , "", "", "");
+
+        when(dataService.getProcessInstanceById(processInstanceId)).thenReturn(processInstanceDesc);
+
+        ImageServiceBase imageServiceBase = new ImageServiceBase(dataService, imageReferenceMap, kieServerRegistry);
+        String processImageStr = imageServiceBase.getActiveProcessImage(containerId, processInstanceId);
+
+        Document svgDocument = readSVG(processImageStr);
+
+        Element subprocessPlusIcon = svgDocument.getElementById(threeScriptNode + "?shapeType=BORDER&renderType=STROKE");
+        String stroke = subprocessPlusIcon.getAttribute("stroke");
+        assertNotNull(stroke);
+        assertEquals("#FF0000", stroke);
+    }
+
+    @Test
+    public void testActiveNodeBorderColor() throws Exception {
+        String containerId = "test-container";
+        String processId = "test-processId";
+        long processInstanceId = 10;
+        String humanTaskOne = "_C8E8A0C7-ECC6-4C28-B1FE-D6DE5999239E";
+        String oneScriptNode = "_BA99E908-87C6-46C9-99ED-901DFCD1A2AA";
+        String twoScriptNode = "_1C98770B-2C62-41F1-AFBF-7B138EE5270E";
+        String threeScriptNode = "_D7C7D018-DC2D-4B97-BEFF-55B50F201103";
+        byte[] byteArray = getInputStreamAsByteArray(ImageServiceBaseTest.class.getResourceAsStream("/BAPL-2159-svg.svg"));
```

Comment:
@bxf12315 would you mind to rename the svg to not to be linked with that BAPL I don't think we need to keep the reference to that BAPL this svg regarding it's just to try the error and active color painting test.